packages/client/src/components/form/ReviewActionComponent.tsx

```diff
@@ -412,7 +412,7 @@ class ReviewActionComponent extends React.Component<
               event: application.event
             })}
             responsive={false}
-            contentHeight={96}
+            contentHeight={115}
```

Comment:
Increasing `contentHeight` of this modal might not work for other cases where content is less. May be we need a prop to auto adjust the modals content height

![image (3)](https://user-images.githubusercontent.com/42269993/147907740-c2a4f9b3-4e50-41a4-8102-bb59ce92a82d.png)
![image (2)](https://user-images.githubusercontent.com/42269993/147907746-d0dc47a5-4d91-41eb-b13f-3c8344d4aed6.png)